backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/util/git/GitUtils.java

```diff
@@ -153,7 +154,7 @@ private static String getNativeGitExecutable() {
         if (exitCode == 0) {
           nativeGitExecutable = executable;
         }
-      } catch (IOException e) {
+      } catch (IOException | IllegalStateException e) {
```

Comment:
Wouldn't it be safer to catch Exception here?

Reply:
Good point, changed to exception.